Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -123,19 +153,36 @@ public final class ViewAnnotationManager {
         }
     }
 
-    /// Update given `UIView` with `ViewAnnotationOptions`.
+    public func removeAll() {
+        let viewsByIdCopy = viewsById
```

Comment:
Do we actually need a copy here?

Reply:
I don't think so. Swift value types should avoid the troubles with mutation during iteration we had in Objective-C

Reply:
Old habits...